spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -51,12 +51,23 @@ interface FungibleTokenPacketDataV2 {
   sender: string
   receiver: string
   memo: string
+  forwardingPath: ForwardingInfo // a struct containing the list of next hops, determining where the tokens must be forwarded next, and the memo for the final hop
+}
+
+interface ForwardingInfo {
+  hops: []Hop
+  memo: string,
+}
```

Comment:
Personally, I feel like the memo in `FungibleTokenPacketData` should cover this, and that we should not include a memo field in this data structure.

Reply:
There would also be overlap between `ForwardingInfo` / `Hop` and the `Trace` type which includes the port id and channel id tuple.

Reply:
> Personally, I feel like the memo in FungibleTokenPacketData should cover this, and that we should not include a memo field in this data structure.

I understand the concern here is preventing unintentional middleware invocation from source chain or intermediary recving chain, but also thought about what if you can rather have a memo attached to each individual hop. 

Spoke about this with @colin-axner and fine to keep this structure as introducing this on a per-hop basis is opening up a big rabbit hole of potential issues.